cluster/src/main/java/org/apache/iotdb/cluster/log/manage/RaftLogManager.java

```diff
@@ -292,15 +292,29 @@ public long getLastLogIndex() {
   public long getTerm(long index) throws EntryCompactedException {
     long dummyIndex = getFirstIndex() - 1;
     if (index < dummyIndex) {
+      // search in disk
```

Comment:
if we manage all the persisted entries(committed or persisted) on the `committedEntryManager`, it's better to move the code in committedEntryManager. but now, the `committedEntryManager` only manager the commit log in memory. and the `LogManager`  managers the `UnCommittedEntryManager` `CommittedEntryManager` and `StableEntryManager`, I think `CommittedEntryManager` is as logs in memory,  `StableEntryManager` is as logs in the disk,  so I think it's ok to leave the code here, the log manager knows where to get the logs

besides, I argue with you that, when get term, we should not care the `isEnableUsePersistLogOnDiskToCatchUp ` parpermeter, if `isEnableRaftLogPersistence` is enabled is ok

Reply:
OK~As the logmanager has been given more functionality, I will try to merge `UnCommittedEntryManager` and `CommittedEntryManager` to a MemoryLogmanager to avoid redundant entry shift. 
I'm OK with here~